react/client/src/filterMenu/FilterMenu.js

```diff
@@ -0,0 +1,25 @@
+import React, {forwardRef} from 'react'
+import PropTypes from 'prop-types'
+
+const FilterMenu = forwardRef(({show, children}, ref)=>{
+    return(
+        <div>
+            {show &&
+                <div className="menu-bg"> 
+                    <div className="menu-main" ref={ref}>
+                        <p>Filter by:</p>
+                        <div className="filters" data-testid='filterSelection'>
+                            {children}
+                        </div>
+                    </div>
+                </div>
+            }
+        </div>
+    )
+})
+FilterMenu.propTypes={
+    show:PropTypes.bool,
+    children:PropTypes.node
+}
+FilterMenu.displayName = FilterMenu
```

Comment:
FYI, displayName needs to be a string. This was causing an error:
`react_devtools_backend.js:6253 Uncaught RangeError: Invalid array length
    at flushPendingEvents (react_devtools_backend.js:6253)
    at react_devtools_backend.js:6935
    at Set.forEach (<anonymous>)
    at Object.flushInitialOperations (react_devtools_backend.js:6915)
    at react_devtools_backend.js:14873
    at <anonymous>:363:34
    at Array.map (<anonymous>)
    at Object.emit (<anonymous>:363:24)
    at attachRenderer (react_devtools_backend.js:14900)
    at react_devtools_backend.js:14912
    at Map.forEach (<anonymous>)
    at initBackend (react_devtools_backend.js:14911)
    at setup (react_devtools_backend.js:11628)
    at welcome (react_devtools_backend.js:11574)`